azurerm/internal/services/appplatform/spring_cloud_service_resource.go

```diff
@@ -133,8 +144,38 @@ func resourceArmSpringCloudService() *schema.Resource {
 				},
 			},
 
+			"trace": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+
+						"app_insight_instrumentation_key": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+					},
+				},
+			},
+
 			"tags": tags.Schema(),
 		},
+
+		// "sku_name" could update from "B0" to "S0"
+		CustomizeDiff: customdiff.ForceNewIfChange("sku_name", func(old, new, meta interface{}) bool {
```

Comment:
it's worth noting this won't work if this field is Interpolated, which it likely will be - so unfortunately this won't catch all users